app/controllers/response_controller.rb

```diff
@@ -181,6 +181,13 @@ def create
     msg = "Your response was successfully saved."
     error_msg = ""
     @response.email
+    begin
+      Response.compare_scores(@response, @questionnaire)
```

Comment:
Does this mean that `compare_score` will be called even for a unsubmitted response?